main.py

```diff
@@ -21,7 +21,7 @@ def parse_args():
                         help="If the argument is provided but empty, the default album path will be used. Optionally, "
                              "provide a path to save the torrent file.")
     parser.add_argument("-c", "--config", help="Tracker configuration name.")
-    parser.add_argument("-f", "--format", choices=["bbc", "md"], default="bbc", help="Table format (bbcode/markdown).")
+    parser.add_argument("-f", "--format", choices=["bbc", "md", "compact_bbc"], default="bbc", help="Table format (bbcode/markdown).")
```

Comment:
I'll update the hint and probably rename `compact_bbc` to `cbbc` so it is less to type